op-faucet/faucet/backend/faucet.go

```diff
@@ -94,6 +114,16 @@ func (f *Faucet) RequestETH(ctx context.Context, request *ftypes.FaucetRequest)
 
 	logger.Info("Sending funds")
 
+	balance, err := f.Balance()
+	if err != nil {
+		logger.Warn("Failed to get balance, optimistically continuing the request")
+	} else {
+		if balance.ToBig().Cmp(request.Amount.ToBig()) < 0 {
+			logger.Info("Insufficient balance", "balance", balance.String(), "amount", request.Amount)
```

Comment:
nit; should this perhaps be Warn or perhaps even Error? We do return an error here after all.

```suggestion
			logger.Error("Insufficient balance", "balance", balance.String(), "amount", request.Amount)
```

Reply:
Whoops